packages/cli/test/integration/credentials.test.ts

```diff
@@ -1,9 +1,11 @@
 import express from 'express';
 import { UserSettings } from 'n8n-core';
+import type { SuperAgentTest } from 'supertest';
 import { Db } from '../../src';
 import { RESPONSE_ERROR_MESSAGES } from '../../src/constants';
 import { CredentialsEntity } from '../../src/databases/entities/CredentialsEntity';
 import type { Role } from '../../src/databases/entities/Role';
+import type { User } from '../../src/databases/entities/User';
```

Comment:
Out of scope: We should consider a lint rule to autosort imports, ideally placing less important type imports grouped at the bottom. If you agree, will track as tech debt.

Reply:
I just use the organize functionality from VSCode which sorts imports alphabetically and groups to library/internal

Reply:
the rules I found were always contradicting so either having the imports sorted alphabetically or the libraries first then internals. Happy to delegate it to ESLint though